lib/Authy/AuthyApi.php

```diff
@@ -41,12 +41,9 @@ class AuthyApi
     public function __construct($api_key, $api_url = "https://api.authy.com")
     {
         $this->rest = new \GuzzleHttp\Client(array(
-            'base_url' => "{$api_url}/protected/json/",
-            'defaults' => array(
-                'headers'       => array('User-Agent' => $this->__getUserAgent() ),
-                'query'         => array('api_key' => $api_key),
```

Comment:
According to this it's supported, did you test it?
https://github.com/guzzle/guzzle/blob/master/src/Client.php#L56